core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -126,27 +125,9 @@ angular.module('oppia').directive('ckEditor4Rte', [
         CKEDITOR.plugins.addExternal(
           'pre', '/extensions/ckeditor_plugins/pre/', 'plugin.js');
 
-        var startupFocusEnabled = true;
-        if (
-          scope.uiConfig() &&
-          scope.uiConfig().startupFocusEnabled !== undefined) {
-          startupFocusEnabled = scope.uiConfig().startupFocusEnabled;
-        }
-        let activeLanguage;
-        let activeLanguageDirection;
-        if (scope.uiConfig()) {
-          activeLanguage = scope.uiConfig().activeLanguage;
-          activeLanguageDirection = scope.uiConfig().activeLanguageDirection;
-        }
-
-        // Initialize CKEditor.
-        var ck = CKEDITOR.inline(<HTMLElement>(el[0].children[0].children[1]), {
-          // Language configs use default language when undefined.
-          language: activeLanguage,
-          contentsLanguage: activeLanguage,
-          contentsLangDirection: activeLanguageDirection,
+        const ckConfig:CKEDITOR.config = {
```

Comment:
I think we normally put a single space after the ":".

/cc @DubeySandeep @sajalasati -- might be worth adding as a linter check.

Reply:
@seanlip Thanks for bringing into our notice! I don't have enough knowledge to define the check properly, so deferring this to @DubeySandeep for adding this in our [list](https://github.com/oppia/oppia/issues/8423).

Reply:
@DubeySandeep I have added the check as an incomplete description there (which you can complete when you get a chance to take a look at it).
Sorry for multiple comments.

Reply:
I added the space on this line.